doc/usecases/elektra_web/UC_setup_instance.md

```diff
@@ -0,0 +1,25 @@
+# Use Case: Setup instance
+
+## Summary
+
+Title: Setup instance
+Scope: Setup
+Level: User Goal
+Actors: User (usually a sysadmin)
+Brief: User sets up a daemon on the instance and connects it to elektra-web.
+
+## Scenarios
+
+Precondition: elektra-web is set up
+Main success scenario: User successfully sets up elektrad on the instance (by
```

Comment:
As already said earlier: too many details, keep the use cases high-level. Remove whats written in () and in the following sentence (API key, ...).